apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md

```diff
@@ -92,10 +84,9 @@ Regarding a full description, please refer to the documentation website. The fol
 **Real-Life Example (Jetstar Airways &amp; Sitecore)**

```

Comment:
Above in #1 and #2 references to Sitecore CDP should really be replaced with Sitecore Personalize.  Especially since this is an XM + P document, but also could give users the belief that CDP is needed for this, but all of this is Personalize.  Lots of documents I've found use CDP/P interchangeably, and sometimes you probably could, but this time you probably shouldn't because it creates confusion.

Reply:
Got it, will replace CDP with Personalize in the first section. I did not see references to CDP in the second use case.